### PR TITLE
Stream funcs

### PR DESCRIPTION
**Pull request by dlydka**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -47,7 +47,15 @@ def unauthorized_access(error):
 
 @app.route("/")
 def index():
-    return render_template("index.html", the_title="Paint Drying")
+    if current_user.is_authenticated:
+        user = db.get_client_by_email(current_user.email)
+        user_data = json.loads(user.to_json())
+        subscription_level = user_data['subscription']['subscription_level']
+        print(subscription_level)
```
Comment (bstrzelecki) [inline src/main.py]: stray print statement

File: src/templates/edit_profile.html [hunk 2]
```diff
@@ -2,88 +2,121 @@
...
+<div class="d-flex flex-column min-vh-100 min-vw-100">
+	<div class="d-flex flex-grow-1 justify-content-center align-items-center">
+		<div class="container-fluid">
+			<div class="row d-flex justify-content-center align-items-center h-100">
+				<div class="col-md-4">
+					<form method="post">
+						{{form.hidden_tag()}}
+						<fieldset>
+							<legend>
+								Update Your Profile
```
Comment (bstrzelecki) [inline src/templates/edit_profile.html]: Mixed language

File: src/main.py (hunk 1 above)
Comment (dlydka) [reply src/main.py]: I already deleted this print

File: src/templates/edit_profile.html (hunk 2 above)
Comment (dlydka) [reply src/templates/edit_profile.html]: This one is in Polish now

**Review by MikolekN — APPROVED** [review]

**Review by bstrzelecki — APPROVED** [review]

**Pull request merged**